platform/kube/inject/testdata/auth.cert-dir.yaml.injected

```diff
@@ -49,10 +49,16 @@ spec:
         securityContext:
           runAsUser: 1337
         volumeMounts:
+        - mountPath: /etc/istio
```

Comment:
These are all virtualized paths. Can you first define the whole list of paths that are needed for istio-on-vm, and we'll align them later? For k8s/docker, these paths carry no meaning.